llvm/test/Transforms/LoopVectorize/RISCV/vplan-vp-call-intrinsics.ll

```diff
@@ -0,0 +1,477 @@
+; REQUIRES: asserts
```

Comment:
Would be good to have not only vplan test but also IR test

Reply:
added